packages/surveys/src/components/general/survey.test.tsx

```
@@ -17,17 +18,25 @@ vi.mock("@/components/general/ending-card", () => ({
-        </button>
-      </div>
-    )),
+    QuestionConditional: vi.fn(
+      ({ onSubmit, onBack }: { onSubmit: (data: any, ttc: any) => void; onBack?: () => void }) => (
+        <div data-testid="question-card">
+          Question Card
+          <button
+            data-testid="submit-button"
+            onClick={() => {
+              onSubmit({ q1: "test answer" }, { q1: 1000 });
+            }}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)